package.json

```diff
@@ -144,13 +144,7 @@
     "typescript": "3.5.3"
   },
   "resolutions": {
-    "@typescript-eslint/parser": "1.11.0",
-    "@typescript-eslint/eslint-plugin": "1.11.0",
-    "handlebars": "4.0.14",
-    "js-yaml": "3.13.1",
-    "hoist-non-react-statics": "3.3.0",
-    "caniuse-lite": "1.0.30000974",
-    "marked": "0.7.0"
+    "hoist-non-react-statics": "3.3.0"
```

Comment:
Can we remove `hoist-non-react-statics`? Some time has passed, maybe we do have a chance to also remove it right now?

Will check.

Reply:
What I understand is that if removed, yarn will resolve to an older 2.x version. So, I kept it.
I think it can be removed once PF4 migration is done and PF3 libraries are no longer needed.